src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateBuildOptionsRule.cs

```diff
@@ -23,7 +23,7 @@ private AddPropertyTransform<CommonCompilerOptions>[] EmitEntryPointTransforms
                 new AddPropertyTransform<CommonCompilerOptions>("OutputType", "Exe",
                     compilerOptions => compilerOptions.EmitEntryPoint != null && compilerOptions.EmitEntryPoint.Value),
                 new AddPropertyTransform<CommonCompilerOptions>("OutputType", "Library",
-                    compilerOptions => compilerOptions.EmitEntryPoint == null || !compilerOptions.EmitEntryPoint.Value)
+                    compilerOptions => compilerOptions.EmitEntryPoint != null && !compilerOptions.EmitEntryPoint.Value)
```

Comment:
So no emitEntryPoint value means no "OutputType" in the .csproj?

---
In reply to: [79760966](https://github.com/dotnet/cli/pull/4216#discussion_r79760966) [](ancestors = 79760966)